### PR TITLE
2019 Project proposals for SixTrack

### PR DESCRIPTION
**Pull request by rdemaria**

### CODE REVIEW

**Comment by aflp91** [comment]
I think the description part from `_gsocprojects/2019/project_sixtrack.md` could be put in the proposal file

File: _gsocproposals/2019/proposal_sixtrackcoll [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+When high energy protons in the LHC are intercepted by the collimator jaws, they undergo scattering due to the interactions of the incoming protons with either the electromagnetic field of the atoms, or the strong potential of the nucleus. Presently, a physics model for the scattering mechanisms is implemented in SixTrack using a Monte-Carlo code. The interactions fall into four categories: nuclear interaction, coulomb scattering, ionization and hard electromagnetic processes (e.g. pair production).
+
+
+## Expected results
+Demonstrate that the C code running on a GPU can run the scattering routines at the same speed or faster than on CPU, while keeping bit-level identical results.
+
+## Mentors
+
+  * [Johann A. Briffa](mailto:johann.briffa@um.edu.mt)
```
Comment (JavierCVilla) [inline _gsocproposals/2019/proposal_sixtrackcoll]: Could you please add an organization description for The University of Malta ?

File: _gsocproposals/2019/proposal_sixtrackcoll [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  * [Riccardo De Maria](mailto:Riccardo.De.Maria@cern.ch)
```
Comment (JavierCVilla) [inline _gsocproposals/2019/proposal_sixtrackcoll]: Could you please update the list of mentors  [`gsoc/2019/mentors.md`](https://github.com/HSF/hsf.github.io/blob/master/gsoc/2019/mentors.md) ?

File: _gsocproposals/2019/proposal_sixtrackcoll (hunk 1 above)
Comment (rdemaria) [reply _gsocproposals/2019/proposal_sixtrackcoll]: I would use Johann CERN credentials (he is registered under EP-UAI) if ok for you.

Comment (JavierCVilla) [reply _gsocproposals/2019/proposal_sixtrackcoll]: That's fine, it is up to the mentors to decide about the affiliation they want to have.

File: _gsocproposals/2019/proposal_sixtrackcoll (hunk 2 above)
Comment (rdemaria) [reply _gsocproposals/2019/proposal_sixtrackcoll]: Done

**Comment by rdemaria** [comment]
> Please, rename your `_gsocprojects/2019/project_sixtrack.md` to `_gsocprojects/2019/project_SixTrack.md`



> I think the description part from `_gsocprojects/2019/project_sixtrack.md` could be put in the proposal file

Done...

**Comment by JavierCVilla** [comment]
Looks good to me, @aflp91 could you have a look?

**Pull request merged**